packages/yarnpkg-core/sources/Configuration.ts

```diff
@@ -184,6 +185,11 @@ export const coreDefinitions: {[coreSettingName: string]: SettingsDefinition} =
     type: SettingsType.ABSOLUTE_PATH,
     default: `./.yarn/cache`,
   },
+  compressionLevel: {
+    description: `Zip files compression level, default: -1, possible values: -1..9, -1 - default middle level compression, 0 - no compression, 9 - max compression`,
+    type: SettingsType.NUMBER,
+    default: -1,
```

Comment:
Do you think it would be better or worse to directly set the `default` to `6` and hardcode the cache to be suffixless on `6` (rather than `-1`)? We don't want the default to change anyway, and `-1` is difficult to understand 🤔

Reply:
The problem is that 6 is not exactly the default compression level for libzip, it is close to use 6's level, but it is not exactly that. It has some extra tweaks for default level, like either compress or store data uncompressed.

Reply:
Well I can map 6 level to default, I can do that